pkg/liqonet/routing/directRouting.go

```diff
@@ -61,32 +61,32 @@ func NewDirectRoutingManager(routingTableID int, podIP string) (Routing, error)
 // An error if something goes wrong and the routes can not be configured.
 func (drm *DirectRoutingManager) EnsureRoutesPerCluster(tep *netv1alpha1.TunnelEndpoint) (bool, error) {
 	var routePodCIDRAdd, routeExternalCIDRAdd, policyRulePodCIDRAdd, policyRuleExternalCIDRAdd, configured bool
-	clusterID := tep.Spec.ClusterID
+	clusterIdentity := tep.Spec.ClusterIdentity
```

Comment:
```suggestion
	clusterID := tep.Spec.ClusterIdentity.ClusterID
```

The clusterIdentity appears to be never accessed directly. Alternatively, you could completely drop the alias. Same below.